packages/web-runtime/src/store/user.ts

```diff
@@ -12,7 +12,8 @@ const getInitialState = () => ({
   quota: null,
   language: null,
   role: null,
-  roles: []
+  roles: [],
+  notif: ''
```

Comment:
Maybe this should be a boolean with `false` default (if it becomes `disableNotifications`).